app/src/main/res/layout/add_edit_page.xml

```diff
@@ -72,168 +94,17 @@
-                android:text="Fear"
-                android:textStyle="bold"
-                android:textSize="15sp"
-                android:layout_gravity="center"/>
-        </LinearLayout>
-        <LinearLayout
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:orientation="vertical"
-            android:layout_marginStart="10dp">
-            <ImageButton
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:id="@+id/sadEmoji"
-                app:srcCompat="@drawable/sad"
-                android:padding="0dp"
-                android:background="@color/white"/>
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="Sad"
-                android:textStyle="bold"
-                android:textSize="15sp"
-                android:layout_gravity="center"/>
-        </LinearLayout>
-        <LinearLayout
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:orientation="vertical"
-            android:layout_marginStart="10dp">
-            <ImageButton
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:id="@+id/shamEmoji"
-                app:srcCompat="@drawable/shameful"
-                android:padding="0dp"
-                android:background="@color/white"/>
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="Shameful"
-                android:textStyle="bold"
-                android:textSize="15sp"
-                android:layout_gravity="center"/>
-        </LinearLayout>
-            <LinearLayout
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:orientation="vertical"
-                android:layout_marginStart="10dp">
-            <ImageButton
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:id="@+id/coolEmoji"
-                app:srcCompat="@drawable/cool"
-                android:padding="0dp"
-                android:background="@color/white"/>
-            <TextView
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="Cool"
-                android:textStyle="bold"
-                android:textSize="15sp"
-                android:layout_gravity="center"/>
-        </LinearLayout>
+            android:orientation="horizontal"
+            android:id="@+id/emojiList">
+            <!--<LinearLayout-->
+                <!--android:layout_width="wrap_content"-->
```

Comment:
Reduced all of the above code into a dynamically loaded wonderland of emotions. Part of the EZ LOAD process.